### DISCUSSION TITLE
Change Memos favicon

### DISCUSSION THREAD

**Discussion by eallion** [usememos/memos#683] [Ideas]
Add this to `Additional script` (中文`自定义脚本`) :
```
var changeIcon = function () {    
    var link = document.head.querySelector("link");
    link.href = "https://w3.some-domain.com/picture.png";
};
changeIcon();
```
Replace the picture url: `https://w3.some-domain.com/picture.png`

![chrome_ssqBR4m2Jw](https://user-images.githubusercontent.com/1698841/205653727-1c2e6dcb-027e-4829-b72f-0217b95e982e.png)

**Comment by lephtHanded** [comment]
`0.12.2` - this does not work for me anymore

**Reply by nodjoy** [reply]
```
setTimeout(() => {
let ico = document.querySelector('link[rel="icon"]');
ico.setAttribute("href", "https://nodjoy.com/img/favicon.ico");
}, 1000);
```
